TLM/CSUtil.Commons/Log.cs

```diff
@@ -46,13 +48,23 @@ private enum LogLevel {
             LogToFile(s, LogLevel.Debug);
         }
 
+        /// <summary>
+        /// Will log only if debug mode, the string is prepared using string.Format
+        /// </summary>
+        /// <param name="format">The text</param>
+        [Conditional("DEBUG")]
+        public static void _DebugFormat(string format, params object[] args) {
+            LogToFile(string.Format(format, args), LogLevel.Debug);
+        }
+
         /// <summary>
         /// Will log only if debug mode is enabled and the condition is true
         /// NOTE: If a lambda contains values from `out` and `ref` scope args,
         /// then you can not use a lambda, instead use `if (cond) { Log._Debug }`
         /// </summary>
         /// <param name="cond">The condition</param>
         /// <param name="s">The function which returns text to log</param>
+        // TODO: Add log thread and replace formatted strings with lists to perform late formatting in that thread
```

Comment:
Create issue ticket for this so we don't lose track of it?

Reply:
It is not really an issue, just an inevitable evolution step if we ever find ourselves drowning in logs. For now there is no action on this.